lib/mdns_lite/application.ex

```diff
@@ -31,6 +29,17 @@ defmodule MdnsLite.Application do
 
       Supervisor.init(children, strategy: :one_for_all)
     end
+
+    defp default_monitor() do
+      excluded_ifnames = Application.get_env(:mdns_lite, :excluded_ifnames, ["lo0", "lo"])
+
+      if Code.ensure_loaded?(VintageNet) do
+        Application.ensure_all_started(:vintage_net)
```

Comment:
`Application.ensure_all_started(:vintage_net)` shouldn't be needed. If it is, there's a problem somewhere else, since `:mdns_lite` depends on `:vintage_net`.

Reply:
I might need help on this then. It was a pain point when trying to run `mix test`. I went through a few resources, tried a bunch of things, but `vintage_net` just never got started and things would fail.

Any help there is totally appreciated. I am prob missing something obvious...

Reply:
Took this out and conditionally added `:vintage_net` to `:extra_applications` in the `application.ex` when its test env. It works, though I don't know if its the "correct" way